meta/main.yml

```diff
@@ -4,11 +4,14 @@ galaxy_info:
   description: OMERO Prometheus exporter
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.6
+  min_ansible_version: 2.3
```

Comment:
Just spotted this. Is this an oversight?

Reply:
I think @pwalczysko got confused with a previous comment on a PR where the ``min_ansible_version`` was set to 2.3 and he bumped it to 2.10
2.10 is required when we have dependencies otherwise we do not have any reason to change it unless we can to set to 2.10 across the various repositories

Reply:
@sbesson yes, oversignt, thank you. @jburel just clarified - if we have dependencies (there are some here in the requirements file), we are going for 2.10.

fixed in https://github.com/ome/ansible-role-omero-prometheus-exporter/pull/14/commits/74cba3446061ce8b7e416cbfaacd819525fc2151